src/store/index.js

```diff
@@ -31,6 +31,7 @@ export default new Vuex.Store({
     searchTerm: '',
     isHiddenContent: true,
     useSdkWallet: false,
+    client: undefined,
```

Comment:
is it used somewhere? Also, I would suggest using `null` instead (as far it is explicitly assigned value).

Reply:
it's not used. Thanks.